balloc/balloc.c

```diff
@@ -54,20 +54,19 @@ enum m0_balloc_allocation_status {
 };
 
 struct balloc_allocation_context {
-	struct m0_balloc	      *bac_ctxt;
-	struct m0_be_tx		      *bac_tx;
+	uint32_t                       bac_criteria;
```

Comment:
What is the reason to re-organize these members?

Reply:
Just to optimize the structure size by putting less sized fields in the beginning and bulkier at the end.